lab_mem/page_replacement/python/algorithms/__init__.py

```diff
@@ -0,0 +1,19 @@
+from .aging import Aging
+from .fifo import FIFO
+from .lru import LRU
+from .nru import NRU
+from .random_alg import Random
+from .second_chance import SecondChance
+
+algorithms = {
+    "aging": Aging(),
+    "fifo": FIFO(),
+    "lru": LRU(),
+    "nru": NRU(),
+    "random": Random(),
+    "second-chance": SecondChance()
+}
+
+def get_algorithm(algorithm_name):
+    assert algorithm_name in algorithms.keys(), "ERROR: Algorithm not found"
```

Comment:
add the `algorithm_name` to the error message

Reply:
Done